src/lib/git/GitRepo.js

```diff
@@ -0,0 +1,65 @@
+import CID from 'cids'
+import { DAGNode } from 'ipld-dag-pb'
+import GitCommit from './GitCommit'
+
+const DEFAULT_HEAD_REF = 'refs/heads/master'
```

Comment:
You can get the default branch from `QmRepo/HEAD`

Reply:
It checks `QmRepo/HEAD` on line 36. If it is not defined then it uses `DEFAULT_HEAD_REF`